mysql-test/suite/sys_vars/t/net_buffer_shrink_interval_basic.test

```diff
@@ -0,0 +1,122 @@
+--source include/restart_mysqld.inc
```

Comment:
I'd split this test in two: in sys_vars I'd leave a skeleton test for variable manipulation, and have functionality test in the main suite.